JobConfig/ensemble/fcl/SamplingInput_sr0.fcl

```diff
@@ -0,0 +1,53 @@
+#include "Production/JobConfig/ensemble/fcl/prolog.fcl"
+
+#livetime: 60.000000
+#rue: 0.000000
+#dem_emin: 75.000000
+#tmin: 450.000000
+#run: 1201.000000
+#nevts: 2605
+
+
+process_name : SI
+
+source : { module_type : SamplingInput
+  inputCommands : @local::Ensemble.InputCommands
+  dropDescendantsOfDroppedBranches: false
+  run : 1201
+  subRun : 0
+  samplingSeed : 1
+  dataSets : { 
+      CeMLL: {
+        fileNames : ["/pnfs/mu2e/tape/phy-sim/dts/mu2e/CeMLeadingLog/MDC2024a_sm4/art/01/1d/dts.mu2e.CeMLeadingLog.MDC2024a_sm4.001210_00000691.art"]
```

Comment:
If this filename is intended as an example please put it in comments. We don't want to hard-code any file names in production scripts.

Reply:
its just an example